src/main/java/com/mraof/minestuck/entity/underling/WyrmEntity.java

```diff
@@ -1,61 +0,0 @@
-package com.mraof.minestuck.entity.underling;
-
-import com.mraof.minestuck.entity.IEntityMultiPart;
-import com.mraof.minestuck.item.crafting.alchemy.GristHelper;
-import com.mraof.minestuck.item.crafting.alchemy.GristSet;
-import net.minecraft.entity.Entity;
-import net.minecraft.entity.EntityType;
-import net.minecraft.util.DamageSource;
-import net.minecraft.world.World;
-
-import java.util.ArrayList;
-
-public class WyrmEntity extends UnderlingEntity implements IEntityMultiPart
-{
-	public ArrayList<Entity> parts = new ArrayList<>();
-	public ArrayList<Integer> partIds = new ArrayList<>();
-	
-	public WyrmEntity(EntityType<? extends WyrmEntity> type, World world)
-	{
-		super(type, world, 0);
-	}
-	
-	@Override
-	public World getWorld()
-	{
-		return this.level;
-	}
-	
-	@Override
-	public boolean attackEntityFromPart(Entity entityPart, DamageSource source, float damage)
-	{
-		return false;
-	}
-	
-	@Override
-	public void updatePartPositions()
-	{
-	}
-	
-	@Override
-	public void addPart(Entity entityPart, int id)
-	{
-	}
-	
-	@Override
-	public void onPartDeath(Entity entityPart, int id)
-	{
-	}
-	
-	@Override
-	public GristSet getGristSpoils()
-	{
-		return GristHelper.generateUnderlingGristDrops(this, damageMap, 87);
-	}
-	
-	@Override
-	protected int getVitalityGel()
-	{
-		return 0;
-	}
-}
```

Comment:
For the moment, leave the wyrm in. You can treat it the same way you treat the giclops but in this situation just give it the basilisk model and animations

Reply:
If you do this, there is actually a piece of consort dialogue that needs removed. 
In ConsortDialogue line 233 "addMessage("wyrm").consort(EnumConsort.TURTLE, EnumConsort.IGUANA);" 
and the line(1889) that goes with it in MinestuckEnUsLanguageProvider "addDialogue("wyrm", "Legends speak of the Wyrm, a giant ivory pillar that radiates joy and happiness and uselessness.");"